fw/http.c

```diff
@@ -4002,8 +4038,7 @@ tfw_h2_adjust_req(TfwHttpReq *req)
 		return r;
 
 	/* First line. */
-	__h2_msg_hdr_val(&ht->tbl[TFW_HTTP_HDR_H2_METHOD], &meth);
-	r = tfw_msg_write(&it, &meth);
+	r = __h2_write_method(req, &it);
```

Comment:
It is not a problem of this patch, but it seems that it is better to rewrite all this code to
```
if (unlikely((r = ...))
    return r;
```

Reply:
Added to https://github.com/tempesta-tech/tempesta/issues/1411